### PR TITLE
Updated the Qdrant tutorial to use the FiftyOne native integration.

### PR DESCRIPTION
**Pull request by lkaneda**
## What changes are proposed in this pull request?

Qdrant tutorial has been updated to use the native integration.

## How is this patch tested? If it is not, please explain why.

Ran cells in the notebook to make sure they worked as expected.

### CODE REVIEW

**Comment by lkaneda** [comment]
I haven't updated the collab notebook or the other resources at the links in the top of the notebook. Do these somehow auto update or do I need to go and change each one?

File: examples/Qdrant_FiftyOne_Recipe.ipynb
[the remark this answers is not in the document]
Comment (jacobmarks) [reply examples/Qdrant_FiftyOne_Recipe.ipynb:1]: Aside from this everything looks great! 👍 LGTM :)

**Review by jacobmarks — CHANGES_REQUESTED** [review]

File: examples/Qdrant_FiftyOne_Recipe.ipynb
[the remark this answers is not in the document]
Comment (lkaneda) [reply examples/Qdrant_FiftyOne_Recipe.ipynb:1]: Weird! I had deleted those cells but it showed up back in here. Thanks for catching that - removing now!

**Review by jacobmarks — APPROVED** [review]
Awesome, LGTM 👍

**Pull request merged**